models/articles.model.js

```diff
@@ -32,3 +39,42 @@ exports.updateArticleVotes = (article_id, votes = 0) => {
       }
     });
 };
+
+exports.selectArticles = topicFilter => {
+  
+  if (topicFilter) {
+    return db
+      .query(
+        `SELECT articles.article_id, title, topic, articles.author, articles.body, articles.created_at, articles.votes, COUNT(comment_id) ::INT AS comment_count
+      FROM articles
+          LEFT JOIN comments ON articles.article_id = comments.article_id
+      WHERE topic = $1
+      GROUP BY articles.article_id
+      ORDER BY articles.created_at`,
+        [topicFilter]
+      )
+      .then(({ rows }) => {
+        const topicsArray = ["paper", "mitch", "cats"]
```

Comment:
I think that this could be checked earlier and the promise rejected before we make the article query to the db. 

The topicsArray should not be hard coded as we may add topics later. You should consider making a query to the db to check whether the topic exists instead.